### PR TITLE
Moved a bunch of Settings from the config file into the GUI

### PR DESCRIPTION
**Pull request by SirMortimer**
Screenshots say most.

This isn't thoroughly tested yet, however, I'm fairly confident. Things to test, specifically:

* Radiation levels, are they what they should be
* CME Storms (days between storms, storm durations)

The other settings should be working OK, as far as I can tell.

This PR will also fix part of #185 (the message notification presets are new, that config didn't exist yet)
Kerbalism
Kerbalism (Advanced)

Also, I'm not sure what to do with these settings, heck I don't even know what they're supposed to do after looking at the code:

`
  // pressure settings
  PressureFactor = 10.0               // pressurized modifier value for vessels below the threshold
  PressureThreshold = 0.9             // level of atmosphere resource that determine pressurized status

  // poisoning settings
  PoisoningFactor = 0.0               // poisoning modifier value for vessels below the threshold
  PoisoningThreshold = 0.02           // level of waste atmosphere resource that determine co2 poisoning status
`

@steamp0rt don't merge just yet, this needs to be tested a bit (which is why I created this PR).

### CODE REVIEW

**Comment by PiezPiedPy** [comment]
Brilliant, saved me a job 😄 
btw Preferences.cs  should be in the System folder, the Modules folder is for PartModules 😉

File: src/Modules/Preferences.cs [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+
+		public override string Section { get { return "Kerbalism"; } }
+
+		public override int SectionOrder { get { return 0; } }
+
+		public override string Title { get { return "Common"; } }
+
+		private static PreferencesBasic instance;
+
+		public static PreferencesBasic Instance
```
Comment (PiezPiedPy) [inline src/Modules/Preferences.cs]: maybe rename this to fetch to fit in better with KSP coding style, no idea why squad thought it a good idea to do such a thing 🤷‍♂️

Comment (PiezPiedPy) [reply src/Modules/Preferences.cs]: same with the other occurrences of Instance

**Comment by SylvainLasnier** [comment]
Awesome!

**Comment by SirMortimer** [comment]
> Brilliant, saved me a job 😄
> btw Preferences.cs should be in the System folder, the Modules folder is for PartModules 😉

yeah I know... didn't bother moving it :P

File: src/Modules/Preferences.cs (hunk 1 above)
Comment (SirMortimer) [reply src/Modules/Preferences.cs]: Actually I stole that bit from kOS. No idea why they named it that way... Instance refers to the singleton pattern I use a lot in different languages.
I'm new to C#, so bear with me...

**Comment by SirMortimer** [comment]
This should also fix #59.

screenshot4

File: src/Modules/Preferences.cs (hunk 1 above)
Comment (PiezPiedPy) [reply src/Modules/Preferences.cs]: Instance should also be used also in C#, its just that Squad uses fetch for some unknown reason

**Comment by SirMortimer** [comment]
Disabling of stock science seems to have no effect. Other than that, I think it's OK

**Comment by steamp0rt** [comment]
Fix the stock science, I guess?

**Comment by steamp0rt** [comment]
As soon as you do, i'll merge this and release 1.9.

**Pull request merged**